src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
-        maybeExecutionStatusStore = Option(new ExecutionStatusStore(binding))
+        maybeExecutionStore = Option(new ExecutionStore(binding))
         maybeSymbolStore = Option(new SymbolStore(binding, actualInputs))
         maybeBackend = Option(backend)
-        executionStatusStore.runnableCalls.foreach { call =>
-          startCallActor(symbolStore, call)
-        }
-        primeMover ! Started
+        // Poll for runnable calls.
+        system.scheduler.schedule(
```

Comment:
I was anticipating the store update on Call completion to be refactored to execute concurrently and possibly not be complete by the time this actor receives its own `CheckExecutionStatus` message, so I sent `CheckExecutionStatus` at intervals to make sure no runnable calls were dropped.  You're right this is unnecessary with the current in-memory synchronous updates to the overshared store data.

Interesting idea about having the CheckExecutionStatus handler set up its own successor.